aries_cloudagent/anoncreds/anoncreds/base_registry.py

```diff
@@ -0,0 +1,59 @@
+"""Base Registry"""
+from abc import abstractmethod
+from ...config.injection_context import InjectionContext
+
+from .anoncreds_objects import (
+    AnonCredsRegistryGetCredentialDefinition,
+    AnonCredsRegistryGetRevocationList,
+    AnonCredsRegistryGetRevocationRegistryDefinition,
+    AnonCredsRegistryGetSchema,
+)
+
+
+class BaseRegistry:
+    """BaseRegistry"""
```

Comment:
Our registries should be expected to also provide a `supported_identifiers` and `method_name` (or something like that) so that the core `AnonCredsRegistry` object can use these values to select which registry should resolve or register a given object + identifier